src/LightSaml/SymfonyBridgeBundle/DependencyInjection/Configuration.php

```diff
@@ -19,18 +20,11 @@ class Configuration implements ConfigurationInterface
     /**
      * Generates the configuration tree builder.
      *
-     * @return \Symfony\Component\Config\Definition\Builder\TreeBuilder The tree builder
      */
-    public function getConfigTreeBuilder()
+    public function getConfigTreeBuilder(): TreeBuilder
     {
         $treeBuilder = new TreeBuilder('light_saml_symfony_bridge');
-
-        if (method_exists($treeBuilder, 'getRootNode')) {
-            $root = $treeBuilder->getRootNode();
-        } else {
-            // BC layer for symfony/config 4.1 and older
-            $root = $treeBuilder->root('light_saml_symfony_bridge');
-        }
+        $root = $treeBuilder->getRootNode();
```

Comment:
To remove the last PHPStan error :
```suggestion
        /** @var ArrayNodeDefinition $root */
        $root = $treeBuilder->getRootNode();
```
(ArrayNodeDefinition is return if the second argument of `TreeBuilder::__construct` is `array`, which is also the default value)

Reply:
I've tried this; got the [following error](https://github.com/phpstan/phpstan/issues/844) and chose the easiest solution  :smile:

Reply:
Arf... That could be a great PR on the phpstan-symfony repo!